api/src/main/java/com/microsoft/gctoolkit/message/JVMEventConsumer.java

```diff
@@ -0,0 +1,4 @@
+package com.microsoft.gctoolkit.message;
+
+public interface JVMEventConsumer {
```

Comment:
From the name of the interface, I expected something like `extends java.util.function.Consumer<JVMEvent>`